### ISSUE TITLE
There should be a way to mark Akismet-flagged comments as non-spam

### ISSUE THREAD

**Issue by dfabulich** [iftechfoundation/ifdb-suggestion-tracker#370]
We had a user recently where Akismet was flagging every single one of a user's comments as spam, even short, obviously harmless messages containing no HTTP links.

I was able to manually resubmit the comment to Akismet as non-spam ("ham"), and that seems to have fixed that one user, but this could easily happen again.

It would be nice to have a clickable link that moderators could use to submit a non-spam comment to Akismet.

**Comment by dfabulich** [comment]
For the record, here's how I did it by hand.

In a local Docker dev environment with full data, I edited `comment.php`. In the call to `setCommentAuthorURL` to use `https://ifdb.org` instead of `get_root_url()` which would normally point to `localhost:8080` in the dev environment.

Then, using the admin tools, I impersonated the flagged user and made an innocent comment as them. I saw in my logs that we were going to email moderators about the spam.

So, before the call to `isCommentSpam()`, I added a line, `$ak->submitHam();` and then deleted and resubmitted the comment. After I did that, the comment sailed through without being flagged as spam.

I think this fixed that user's account, and it'd be cool to make this a feature that any moderator could use.
